runtime/parachains/src/paras_inherent.rs

```diff
@@ -404,12 +407,7 @@ pub mod pallet {
 			// And track that we've finished processing the inherent for this block.
 			Included::<T>::set(Some(()));
 
-			Ok(Some(paras_inherent_total_weight::<T>(
-				backed_candidates_len,
-				bitfields_len,
-				disputes_len,
-			))
-			.into())
+			Ok(Some(total_weight).into())
```

Comment:
This isn't necessary. With the current logic this will be the same weight as what we give to `pallet::weight(`. We can just return `Ok()` here and the macro will use the weight from the `pallet::weight(` annotation. Returning the weight at the end of the call is only helpful when we used less weight than expected.

Reply:
I tried to return Ok(), but the compiler would not allow it